### PR TITLE
Add "Open in MusicBrainz" link in ListenCard actions

### PR DESCRIPTION
**Pull request by MonkeyDo**
If the recording MBID is available , let's offer the option to open it in MB (even though the title of the recording in the ListenCard will already be a link.
This offers a more explicit alternative and puts the name of MusicBrainz forward.

image

### CODE REVIEW

File: listenbrainz/webserver/static/js/src/listens/ListenCard.tsx [hunk 1]
```diff
@@ -296,6 +297,15 @@ export default class ListenCard extends React.Component<
                 className="dropdown-menu dropdown-menu-right"
                 aria-labelledby="listenControlsDropdown"
               >
+                {recordingMBID ? (
+                  <a
```
[the remark this answers is not in the document]
Comment (akshaaatt) [reply listenbrainz/webserver/static/js/src/listens/ListenCard.tsx:301]: Screenshot from 2021-11-02 12-22-46

This works good for me.

**Review by akshaaatt — CHANGES_REQUESTED** [review]
Thanks for the work here! A few changes and we'll be good to go

File: listenbrainz/webserver/static/js/src/listens/ListenCard.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (MonkeyDo) [reply listenbrainz/webserver/static/js/src/listens/ListenCard.tsx:301]: I thought about that at first, but it's an anti-pattern.
We want to ensure we have clickable anchor elements (<a>) to preserve standard browser functionality such as showing visited link, showing a preview of the link on hover, as well as for screen readers and other assistive technology.

What I can do however is to modify the  ListenControl component to accept an optional `link` prop, and render an <a> tag instead when that prop is used.

**Review by akshaaatt — APPROVED** [review]
Works great. Thanks! :100:

**Pull request merged**